### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by trepmal**
Sourced from https://wpvip.com/documentation/vip-go/geo-targeting-on-vip-go/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+VIP Go allows you to differentiate visitors from different countries, tailoring the content you serve to your visitors on a country by country basis, while still keeping the benefits of our fast caching infrastructure.
+
+Adding geo targeting to your VIP Go code takes two parts:
+
+1. Loading and configuring the VIP Go Geo Uniques plugin
+2. Serving geo targeted content
+
+## Loading the VIP Go Geo Uniques plugin
+
+As the VIP Go Geo Uniques plugin requires configuration in code, we recommend loading this plugin using the wpcom_vip_load_plugin function.
```
Comment (rinatkhaziev) [inline README.md]:
```suggestion
As the VIP Go Geo Uniques plugin requires configuration in code, we recommend loading this plugin using the `wpcom_vip_load_plugin` function.
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+Download the plugin from the GitHub repo (https://github.com/Automattic/vip-go-geo-uniques), add it to your `plugins` folder, and then activate it for your site:
+	
+```
+// Load the VIP Go Geo Uniques plugin
+wpcom_vip_load_plugin( 'vip-go-geo-uniques' );
+```
+
+## Configure the plugin
+
+To configure the plugin, you must set a default location using the VIP_Go_Geo_Uniques::set_default_location method. For example, to set the United States are the default:
```
Comment (rinatkhaziev) [inline README.md]:
```suggestion
To configure the plugin, you must set a default location using the `VIP_Go_Geo_Uniques::set_default_location` method. For example, to set the United States are the default:
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+```
+VIP_Go_Geo_Uniques::set_default_location( 'US' );
+```
+
+Next, configure each region using theVIP_Go_Geo_Uniques::add_location method. If a region is not set, visits from that region will use the default location. As an example, to create a geo unique region of the United Kingdom you would call:
```
Comment (rinatkhaziev) [inline README.md]:
```suggestion
Next, configure each region using the `VIP_Go_Geo_Uniques::add_location method`. If a region is not set, visits from that region will use the default location. As an example, to create a geo unique region of the United Kingdom you would call:
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+This is because when a request comes via a reverse proxy, the remote IP address is that of the reverse proxy. The load balancers on VIP Go will assign the country code to the reverse proxy IP, and not the IP address of the end user; sometimes this may by chance be the right country for the end user, but many times it will not be.
+
+## Unexpected country codes
+
+Our country codes are supplied via the Maxmind GeoIP database, which implements special “pseudo country codes” for particular situations:
+
+> Please note that “EU” and “AP” codes are only used when a specific country code has not been designated (see FAQ). Blocking or re-directing by “EU” or “AP” will only affect a small portion of IP addresses. Instead, you should list the countries you want to block/re-direct individually.
+> – GeoIP and ISO 3166 Country Codes
+
+At time of writing, the GeoIP pseudo  country codes are:
```
Comment (rinatkhaziev) [inline README.md]:
```suggestion
At time of writing, the GeoIP pseudo country codes are:
```

**Review by rinatkhaziev — CHANGES_REQUESTED** [review]
Left some minor suggestions, otherwise looks good!

**Comment by gamebits** [comment]
Somewhere in the header or footer, can we [link back to wpvip.com](https://wpvip.com/documentation/vip-go/geo-targeting-on-vip-go/)? I suspect the documentation is more likely to be maintained there than in the repo. 🤔

**Review by rinatkhaziev — APPROVED** [review]

**Pull request merged**